apps/changelog/classes/change_log.py

```diff
@@ -0,0 +1,79 @@
+# HTK Imports
+from htk.apps.changelog.constants.defaults import (
+    HTK_CHANGELOG_SLACK_CHANNEL_RELEASES,
+    HTK_CHANGELOG_SLACK_FULL_LOG_MESSAGE,
+)
+
+
+# isort: off
+
+
+class ChangeLog:
```

Comment:
What if we used a `namedtuple` to make the code a bit more concise, instead of the `__init__` methods which pretty much just assign every argument to `self.<arg>` ?

https://docs.python.org/3/library/collections.html#collections.namedtuple

(Suggesting `namedtuple` instead of `dataclasses.dataclass` since HTK still has a good number of users on Python 2)

Reply:
Removed and added `namedtuple`